test/common/quic/envoy_quic_proof_verifier_test.cc

```diff
@@ -1,7 +1,8 @@
 #include <algorithm>
 #include <memory>
 
-#include "extensions/quic_listeners/quiche/envoy_quic_proof_verifier.h"
+#include "common/quic/envoy_quic_proof_verifier.h"
```

Comment:
so I'm all for QUIC being built in, but why the big file move?  If we have TLS in extensions why not QUIC?
(I'm also happy with everything core being in core, but we have a bunch of other exceptions so I'd like to make sure we have a plan and maybe file a follow up issue for others)

Reply:
The other things that are "core extensions" fall into 2 buckets today:
1) TLS which is an extension because technically there is an openssl port out there.
2) Extensions that actually are true extensions in terms of documentation, proto build, etc. but we force into the build because they are needed for core functionality (like admin server, etc.).

IMO QUIC doesn't fall into either of those categories which is why I moved the code. I just can't see any situation in which we will override the QUICHE code realistically.